examples/topic_extraction_example.py

```diff
@@ -0,0 +1,98 @@
+"""Copyright 2019 REV
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+
+import time
+from rev_ai import topic_extraction_client, apiclient
+
+
+# String containing your access token
+access_token = "<your_access_token>"
+
+# Create your api client
+client = topic_extraction_client.TopicExtractionClient(access_token)
+
+# Submit a job with whatever text you want by changing this input
+text = "<input_text>"
+job = client.submit_job_from_text(text,
+                                  metadata=None,
+                                  callback_url=None,
+                                  delete_after_seconds=None,
+                                  language=None)
+
+# If you'd like to submit the transcript of an existing transcription job you can do so by
+# uncommenting the lines below
+#
+# async_job_id = "your_job_id"
+# async_api_client = apiclient.RevAiAPIClient(access_token)
+# transcript = async_api_client.get_transcript_object(async_job_id)
+# transcript_json = transcript
+# job = client.submit_job_from_transcript(transcript_json,
+#                                         metadata=None,
+#                                         callback_url=None,
+#                                         delete_after_seconds=None,
+#                                         language=None)
+
+print("Submitted Job")
+
+while True:
+
```

Comment:
is there a reason to have so many newlines? double-newlines above

Reply:
I removed some that seemed unnecessary